src/commonMain/kotlin/character/Stats.kt

```diff
@@ -99,6 +100,10 @@ data class Stats(
     var arcaneDamageMultiplier: Double = 1.0,
     var petDamageMultiplier: Double = 1.0,
 
+    // Spell school damage taken increases
+    var spellDamageTakenMultiplier: Double = 1.0,
```

Comment:
I had thought about this one quite a bit in both trying to simulate the damage (spell / shadow damage taken are real aura values in game, see [example](https://tbc.wowhead.com/spell=15258/shadow-vulnerability) where `Apply Aura: Mod % Damage Taken (Shadow)`) and not expand damage roll methods too much (Since they had too many parameters as a code smell). I agree the individual calculation felt weird for direct damage spells and should use a damage calculator within `Spell` to be consistent across both direct and DoTs going forward.

I'll add the base spell damage / canResist pieces for snap shots and I'll integrate all damage calcs into base / attack roll going forward. In keeping with spell damage taken stat to simulate, did you want me to go ahead and add `spellDamageTakenMultiplier` to Misery and Curse of Elements here and add those as raid debuffs instead so the spells are consistent or should we revisit this later and assume it's a part of base spell damage for now? In terms of damage on dots there isn't any change as it's just a multiplier regardless, but at least for direct damage it would correctly be applied

Reply:
> did you want me to go ahead and add spellDamageTakenMultiplier to Misery and Curse of Elements here and add those as raid debuffs instead so the spells are consistent or should we revisit this later and assume it's a part of base spell damage for now? 

It's equivalent for single-target, and there's a lot of other work required before multi-target and complete raid sims can be done - I think it's best to punt on that for now.